packages/manager/apps/dedicated/client/app/ip/ip/ip-ip.controller.js

```diff
@@ -647,10 +647,26 @@ angular
         set(ipBlock, 'refreshing', true);
         IpReverse.updateReverse(ipBlock, ip.ip, ip.reverseEditValue).then(
           () => {
-            $rootScope.$broadcast('ips.table.refreshBlock', ipBlock);
-            Alerter.success(
-              $translate.instant('ip_table_manage_reverse_success'),
-            );
+            if (!ipBlock.isUniq) {
```

Comment:
it's better to avoid double negation : if (ipBlock.isUniq) { ...

Reply:
sorry not a double negation but it's common practice to put the "true" block in the if and the "false" block in the else. it has better readability

Reply:
👍  and `else` is unnecessary.
```js
if (ipBlock.isUniq) {
  return loadService(ipBlock.service) // ...
}

$rootScope.$broadcast('ips.table.refreshBlock', ipBlock);
Alerter.success($translate.instant('ip_table_manage_reverse_success'));
return null;
```